pkgs/applications/networking/cluster/kpt/default.nix

```
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "kpt";
-  version = "0.39.3";
+  version = "1.0.0-beta.49";
 
   src = fetchFromGitHub {
-    owner = "GoogleContainerTools";
+    owner = "kptdev";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-vidrKfmP0Lw6EYYufLDxh3ROOJ3hPIusDTI/Hr73NYM=";
+    hash = "sha256-whqA1ACN7vXSDWVi55/fJKnuBrDCIdOpyDS8KNrhZq0=";
   };
 
-  vendorHash = "sha256-CoXlUX9hkP8gijA/vg19AS9030w95A2oKGD1wjzO8ak=";
+  vendorHash = "sha256-NQ/JqXokNmr8GlIhqTJb0JFyU2mAEXO+2y5vI79TuX4=";
```

Vendor hash for `1.0.0-beta.55`

```suggestion
  vendorHash = "sha256-2jJCvBtTiIYmpxA92p8eZnKl1UO74pKr1YFRH14keZY=";
```